pkg/synopsysctl/cmd_deploy.go

```diff
@@ -141,11 +125,6 @@ func init() {
 	deployCmd.Flags().StringVarP(&synopsysOperatorImage, "synopsys-operator-image", "i", synopsysOperatorImage, "synopsys operator image URL")
 	deployCmd.Flags().StringVarP(&exposeMetrics, "expose-metrics", "m", exposeMetrics, "expose the Synopsys-Operator's metrics application. possible values are [NODEPORT/LOADBALANCER/OPENSHIFT]")
 	deployCmd.Flags().StringVarP(&metricsImage, "metrics-image", "k", metricsImage, "image URL for the Synopsys-Operator's metrics pod")
-	deployCmd.Flags().StringVar(&deploySecretType, "secret-type", deploySecretType, "type of kubernetes secret to store the postgres and blackduck credentials")
-	deployCmd.Flags().StringVarP(&adminPassword, "admin-password", "a", adminPassword, "postgres admin password")
-	deployCmd.Flags().StringVarP(&postgresPassword, "postgres-password", "p", postgresPassword, "postgres password")
-	deployCmd.Flags().StringVarP(&userPassword, "user-password", "u", userPassword, "postgres user password")
-	deployCmd.Flags().StringVarP(&blackduckPassword, "blackduck-password", "b", blackduckPassword, "blackduck password for 'sysadmin' account")
```

Comment:
@msenmurugan I'm confused about `blackduck-password`.  Is it supposed to be at the hub level or the operator level?  The comment here indicates its at hub level, but how it's used as `HUB_PASSWORD` for opssights, it seems we should keep it at operator level?

Reply:
[Looks like somebody tried to previously add it](https://github.com/blackducksoftware/synopsys-operator/blob/master/pkg/opssight/ctl_opssight.go#L99)